src/component/Roadmap.js

```diff
@@ -11,11 +11,65 @@ export class Roadmap {
   }
 
   getTasks() {
-    return this.tasks;
+    return this._sortItems(this.tasks);
   }
 
   getPeople() {
-    return this.people;
+    return this._sortItems(this.people);
+  }
```

Comment:
`sortItems` accepts tasks or peoples, but it's implicitly.

Should create `AbstractItem` class.
Tasks extends AbstractItem.
Person extends AbstractItem.

What do you think?